webpack/build.babel.js

```diff
@@ -24,9 +25,12 @@ export default merge(base, {
         },
         extractComments: false,
       }),
-      new OptimizeCSSAssetsPlugin({}),
+      new CSSMinimizerPlugin(),
     ],
   },
+  performance: {
+    hints: false,
+  },
```

Comment:
I took the liberty of removing the warnings related to asset size displayed on `yarn export` / `yarn preview`, since the bundle size is already visible and the end-user doesn't have any pointers on how to reduce that value.

I'm open to discussing it there are other opinions / options.